packages/mdc-textfield/README.md

```diff
@@ -176,24 +185,55 @@ UX for client-side form field validation.
 </p>
 ```
 
-### Multi-line - With Javascript
+### Leading and Trailing Icons
+Leading and trailing icons can be added to MDC Textfields as visual indicators
+as well as interaction targets. To do so, add the relevant classnames to the root element, add
+an `i` element with your preferred icon, and give it either a `mdc-textfield__leading-icon` or
```

Comment:
These docs are out of date with what's in the demo examples now, which uses just `__icon` in both cases on the icon element.